docs/verifiable-payid.md

```diff
@@ -4,6 +4,8 @@ title: Getting Started With Verifiable PayID
 sidebar_label: Verifiable PayID
 ---
 
+**NOTES** - This doc will be cut up to have both a tutorial and a concepts section.
```

Comment:
Make sense. The bullet points on lines 11-13 sound like potential tutorials!

Reply:
Cut and shuffled.